simple/stats/nl.py

```diff
@@ -12,38 +12,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from dataclasses import dataclass
+from dataclasses import field
 import logging
 
 import pandas as pd
-from stats.data import StatVar
+from stats.data import Triple
+import stats.schema_constants as sc
 from util.filehandler import FileHandler
 
 _DCID_COL = "dcid"
 _SENTENCE_COL = "sentence"
 _SENTENCE_SEPARATOR = ";"
 
 
-def generate_sv_sentences(svs: list[StatVar], sentences_fh: FileHandler):
-  """Generates sentences based on the name, description and NL sentences of the specified SVs.
+def generate_nl_sentences(triples: list[Triple], sentences_fh: FileHandler):
```

Comment:
generate_search_descriptions?

Reply:
This generates an intermediary sentences.csv which is used to generate embeddings. So left it as-is.